src/class-theme.php

```diff
@@ -35,7 +35,7 @@
  *
  * @since 1.0.0
  */
-class Main extends Lib_Core {
+class Theme extends Lib_Core {
```

Comment:
agreed this does looks way cleaner. But we would like to use this main entrypoint to the application as "Main" because we are using the same setup for our https://github.com/infinum/eightshift-boilerplate-plugin so that the the naming is the same. 

Can you reverse this back to Main?